Description: Remove dependency to loom in tests
Forwarded: not-needed
Author: Antonin Delpeuch <antonin@delpeuch.eu>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -45,10 +45,6 @@ name = "ecow"
 path = "src/lib.rs"
 
 [[test]]
-name = "loom"
-path = "tests/loom.rs"
-
-[[test]]
 name = "tests"
 path = "tests/tests.rs"
 
@@ -60,7 +56,3 @@ path = "tests/ub_guards.rs"
 version = "1.0"
 optional = true
 default-features = false
-
-[target."cfg(loom)".dependencies.loom]
-version = "0.7"
-optional = true
